frontend/src/components/widgets/WidgetRow.vue

```diff
@@ -17,13 +18,16 @@
 			<OptionalInput v-if="component.kind === 'OptionalInput'" v-bind="component.props" @update:checked="(value: boolean) => updateLayout(component.widget_id, value)" />
 			<RadioInput v-if="component.kind === 'RadioInput'" v-bind="component.props" @update:selectedIndex="(value: number) => updateLayout(component.widget_id, value)" />
 			<Separator v-if="component.kind === 'Separator'" v-bind="component.props" />
+			<TextLabel v-if="component.kind === 'TextLabel'" v-bind="component.props">{{ component.props.value }}</TextLabel>
+			<IconLabel v-if="component.kind === 'IconLabel'" v-bind="component.props" />
+			<NumberInput v-if="component.kind === 'TextInput'" v-bind="component.props" @update:value="(value: number) => updateLayout(component.widget_id, value)" />
```

Comment:
You are using NumberInput for text input? Also updateLayout should be `value: number`

Reply:
This was a placeholder until I merged my TextInput a few minutes ago.

Reply:
Ah ok, somewhat confused by that :)

Reply:
It's hooked up now!!